crates/rpc/rpc/src/eth/error.rs

```diff
@@ -492,8 +492,11 @@ impl RevertError {
 impl std::fmt::Display for RevertError {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.write_str("execution reverted")?;
-        if let Some(reason) = self.output.as_ref().and_then(decode_revert_reason) {
-            write!(f, ": {reason}")?;
+        if let Some(ref output) = self.output {
```

Comment:
`as_ref().and_then(decode_revert_reason)` should still work here no?

Reply:
yes, with just a little map on it